src/main/java/com/minecolonies/coremod/client/gui/containers/WindowCitizenInventory.java

```diff
@@ -13,6 +13,11 @@
  */
 public class WindowCitizenInventory extends ContainerScreen<ContainerCitizenInventory>
 {
+    /**
+     * Texture res loc.
+     */
+    private static final ResourceLocation TEXT = new ResourceLocation(Constants.MOD_ID, "textures/gui/generic_108.png");
```

Comment:
Can we pick another name? this name makes me think the texture is an image of "lore ipsum sit amet" in some fancy font.